lib/buildFactory.js

```diff
@@ -168,7 +169,7 @@ class BuildFactory extends BaseFactory {
                     ]).then(([setup, teardown]) => {
                         modelConfig.steps = setup.concat(modelConfig.steps, teardown);
                         // Launcher is hardcoded to do some business in sd-setup-launcher
-                        modelConfig.steps.unshift({ name: 'sd-setup-launcher' });
+                        modelConfig.steps.unshift({ name: 'sd-setup-launcher', alwaysRun: true });
```

Comment:
This is the first step, so I guess it will alwaysRun without the flag.. And I think we are missing the `alwaysRun: true` flag for the teardown bookends?